src/Utils/Logging/Config.hs

```diff
@@ -197,7 +197,8 @@ validateBackendConfig o = do
         validateHandleConfig $ _backendConfigHandle o
         case (_backendConfigHandle o, _backendConfigColor o) of
             (FileHandle _, ColorTrue) ->
-                tell ["log messages are formatted using ANSI color escape codes but are written to a file"]
+              throwError
+              "log messages are formatted using ANSI color escape codes but are written to a file"
```

Comment:
You you remove this change? It is rare but happens that one wants to write the logs to a file including the ANSI codes. `tell` just prints a warning, but doesn't abort the program.

Reply:
I reverted it in the latest commit.